packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/delete-container/delete-container.controller.js

```diff
@@ -29,12 +31,18 @@ export default class ColdArchiveContainersDeleteContainerController {
     );
   }
 
+  getDocumentationUrl() {
+    return MANAGE_ARCHIVE_DOC_LINK[
+      this.coreConfig.getUserLanguage().toUpperCase()
+    ];
```

Comment:
**question:** I see a default link in the constants file. How is it used?

Reply:
The default link is used in the niche case where the `getUserLanguage()` returns a new language key and this constant array is not updated yet.